iform.hostsite_api.inc

```diff
@@ -413,4 +413,21 @@ function _iform_get_alternative_user_id($account, $default) {
       return $default;
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Can these methods be commented please Gary? Also Drupal linting standards put the opening brace on previous line, not on line on itself. Sorry - being fussy!